### ISSUE TITLE
int2uint implicit conversions not support at #extension GL_ARB_gpu_shader5.

### ISSUE THREAD

**Issue by ZhiqianXia** [KhronosGroup/glslang#2699]
**shader**

```
#version 150
#extension GL_ARB_gpu_shader5 : require

uniform int u1;
uniform int u2;
out   vec4 result;
void main()
{
    uint v = 0;
    v = uint(u2) - u1;  // implicit conversions
}
```

ref:
https://www.khronos.org/registry/OpenGL/extensions/ARB/ARB_gpu_shader5.txt

**Comment by ZhiqianXia** [comment]
PR: #2700

**Comment by ZhiqianXia** [comment]
@greg-lunarg Could you please review the PR?

**Comment by greg-lunarg** [comment]
Fixed by #2700
